change/@fluentui-react-text-8e9536d4-ec33-44d4-abc3-f2cc51633f53.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "chore(Text): Add docs for each prop and new typography story",
+  "packageName": "@fluentui/react-text",
+  "email": "lingfangao@hotmail.com",
+  "dependentChangeType": "patch"
```

Comment:
why is this a patch ? there are no changes to package API surface

Reply:
Ah good catch, that's a mistake

Reply:
```suggestion
  "dependentChangeType": "none"
```